content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+When you're making a game in Bevy, you will have a single `App`, which stores and handles all of your game logic and data.
```

Comment:
Unless referring to a specific "game", I think we should refer to arbitary Bevy programs as "apps". So this should read:
"When you're making an app in Bevy, you will have a single `App`, which stores and handles ..."